### PR TITLE
Ignore annotation names

### LINKED ISSUE #1603: False positive for NonFullyQualifiedClassName
I have the following php docblock

```
    /**
     * @apiParam (GET) {Number} [page] Page number
     * /

```

Getting:
```
563 | ERROR | [x] Class name \App\Controller\GET in @apiParam
     |       |     should be referenced via a fully qualified
     |       |     name.
     |       |     (SlevomatCodingStandard.Namespaces.FullyQualifiedClassNameInAnnotation.NonFullyQualifiedClassName)
```

this happened after updating to 8.13.4

### PR DESCRIPTION
**Pull request by gemal**
fixes #1603

### CODE REVIEW

File: SlevomatCodingStandard/Sniffs/Namespaces/FullyQualifiedClassNameInAnnotationSniff.php [hunk 1]
```diff
@@ -158,4 +169,14 @@ public function process(File $phpcsFile, $docCommentOpenPointer): void
 		}
 	}
 
+	/**
+	 * @return list<string>
+	 */
+	private function getIgnoredAnnotationNames(): array
+	{
+		$this->ignoredAnnotationNames = SniffSettingsHelper::normalizeArray($this->ignoredAnnotationNames);
```
Comment (kukulich) [inline SlevomatCodingStandard/Sniffs/Namespaces/FullyQualifiedClassNameInAnnotationSniff.php:177]: Please normalize the array in `process` method, see https://github.com/slevomat/coding-standard/blob/master/SlevomatCodingStandard/Sniffs/Classes/AbstractPropertyConstantAndEnumCaseSpacing.php#L58-L61

File: SlevomatCodingStandard/Sniffs/Namespaces/FullyQualifiedClassNameInAnnotationSniff.php [hunk 2]
```diff
@@ -23,6 +25,9 @@ class FullyQualifiedClassNameInAnnotationSniff implements Sniff
 
 	public const CODE_NON_FULLY_QUALIFIED_CLASS_NAME = 'NonFullyQualifiedClassName';
 
+	/** @var list<string> */
+	public $ignoredAnnotationNames = [];
```
Comment (kukulich) [inline SlevomatCodingStandard/Sniffs/Namespaces/FullyQualifiedClassNameInAnnotationSniff.php:29]: Please add the option to sniff documentation.

**Review by kukulich — CHANGES_REQUESTED** [review]

File: SlevomatCodingStandard/Sniffs/Namespaces/FullyQualifiedClassNameInAnnotationSniff.php (hunk 2 above)
Comment (gemal) [reply SlevomatCodingStandard/Sniffs/Namespaces/FullyQualifiedClassNameInAnnotationSniff.php:29]: you mean in `namespaces.md` file?

File: doc/namespaces.md [hunk 3]
```diff
@@ -180,3 +180,8 @@ Sniff provides the following settings:
 * `searchAnnotations` (defaults to `false`): enables searching for class names in annotations.
 * `ignoredAnnotationNames`: case-sensitive list of annotation names that the sniff should ignore (only the name is ignored, annotation content is still searched). Useful for name collisions like `@testCase` annotation and `TestCase` class.
 * `ignoredAnnotations`: case-sensitive list of annotation names that the sniff ignore completely (both name and content are ignored). Useful for name collisions like `@group Cache` annotation and `Cache` class.
+
+#### SlevomatCodingStandard.Namespaces.FullyQualifiedClassNameInAnnotation 🔧
```
Comment (kukulich) [inline doc/namespaces.md:184]: The sniff is already documented above.

**Comment by kukulich** [comment]
Please squash it to one commit and change the commit message like this: https://github.com/slevomat/coding-standard/commit/3200c0bbede09d1a6c51986bdfda3efcfa8eaff0

**Review by kukulich — CHANGES_REQUESTED** [review]
Please squash it to one commit and change the commit message like this: https://github.com/slevomat/coding-standard/commit/3200c0bbede09d1a6c51986bdfda3efcfa8eaff0

**Comment by gemal** [comment]
> Please squash it to one commit and change the commit message like this: [3200c0b](https://github.com/slevomat/coding-standard/commit/3200c0bbede09d1a6c51986bdfda3efcfa8eaff0)

not sure how to do this. I'm using github desktop as a client

**Comment by gemal** [comment]
> > Please squash it to one commit and change the commit message like this: [3200c0b](https://github.com/slevomat/coding-standard/commit/3200c0bbede09d1a6c51986bdfda3efcfa8eaff0)
> 
> not sure how to do this. I'm using github desktop as a client

better now?

**Comment by gemal** [comment]
@kukulich Thank you for nice and fast guide for me to contribute. An excellent example of open source!

**Pull request merged**